resources/statements.ts

```diff
@@ -17,12 +17,13 @@ export class Statments extends BaseResource {
         return this.httpGet<UnitResponse<Statement[]>>("", { params: parameters })
     }
 
-    public get(statementId: string, customerId?: string): Promise<string> {
+    public get(statementId: string, customerId?: string, isPDF: boolean = false): Promise<string> {
```

Comment:
Went with the quick solution, but could change `isPDF` to an enum somewhere else, containing something like:
enum FileType {
   HTML: 'html',
   PDF: 'pdf',
}